pages/partials/questionNavSideButton.ejs

```diff
@@ -0,0 +1,22 @@
+<div class="text-white mb-3">
+  <a id="<%= button.id %>"
+    <% if (question.id && !question.sequence_locked) { %>
+
+      class="btn btn-primary"
+      href="<%= urlPrefix %>/instance_question/<%= question.id %>/"
+    <% } else { %>
+      class="btn <% if (question.sequence_locked) { %>btn-secondary pl-sequence-locked <% } else { %> btn-primary disabled <% } %>"
+      <% if (question.sequence_locked) { %>
+        tabindex="0" role="button"
+        data-toggle="popover" data-trigger="focus" data-container="body"
+        data-html="true"
+        data-content="You must score at least <b><%= unlock_score %>%</b> on this question to unlock the next. If you run out of attempts, the next question will unlock automatically."
```

Comment:
It looks like the variable `unlock_score` here should actually be called `unlock_score_perc`? Confusingly, we have some variables in PL that store scores on a [0,1] scale (which we call `score`) and some that use a [0,100] scale (which we call `score_perc`). To try and maintain at least a little sanity, it's crucial to use `score` or `score_perc` appropriately. As well as fixing this variable name, can you check whether there are any others in this PR that need fixing?

Reply:
Done. There aren't any instances of `AdvanceScore` or `advance_score` that aren't followed immediately by `perc` in the codebase anymore.